test/SessionKeyTest.ts

```diff
@@ -271,7 +271,7 @@ class SessionTester {
     };
 
     const signedTransaction = await this.sessionAccount.signTransaction(this.aaTransaction);
-    await expect(provider.broadcastTransaction(signedTransaction)).to.be.reverted;
+    await expect (provider.broadcastTransaction(signedTransaction)).to.be.reverted;
```

Comment:
huh, how did prettier not complain about the extra space

Reply:
hah, this is the part that era test node broke. Easy cleanup!